python/cuml/solvers/qn.pyx

```diff
@@ -160,9 +160,9 @@ class QN(Base):
         # Note: for now, the coefficients also include the intercept in the
         # last position if fit_intercept=True
         print("Coefficients:")
-        print(solver.coef_.copy_to_host())
+        print(solver.coef_)
```

Comment:
Should this use the `solver_model` improvements made in PR#2587?

Reply:
what do you mean? It uses the changes from that PR automatically, and QN is the solver so it doesn't have a solver_model